persistence-testkit/src/test/scala/org/apache/pekko/persistence/testkit/scaladsl/RuntimeJournalsSpec.scala

```diff
@@ -0,0 +1,123 @@
+package org.apache.pekko.persistence.testkit.scaladsl
+
+import com.typesafe.config.ConfigFactory
+import org.apache.pekko.Done
```

Comment:
we prefer to import 'org.apache.pekko' as 1 import and all other imports for pekko classes can skip the 'org.apache' part.